lib/src/main/java/com/springboot/webpush/common/api/OnActionClick.java

```diff
@@ -1,14 +1,9 @@
 package com.springboot.webpush.common.api;
 
+import com.springboot.webpush.common.api.types.OnActionClickOperation;
+
 /**
- * The {@link OnActionClick} class is used to control how push messages are to behave. Valid operations are:<br>
- * openWindow Opens a new tab at the specified URL.<br>
- * focusLastFocusedOrOpen Focuses the last focused client. If there is no client open, then it opens a new tab at the
- * specified URL. (url is optional)<br>
- * navigateLastFocusedOrOpen Focuses the last focused client and navigates it to the specified URL. If there is no
- * client open, then it opens a new tab at the specified URL.<br>
- * sendRequest Send a simple GET request to the specified URL.<br>
- *
+ * The {@link OnActionClick} class is used to control how push messages are to behave.
  */
 public class OnActionClick {
     private final String operation;
```

Comment:
FYI --  I did not change the INTERNAL property to an enumeration -- because this OBJECT was being sent to the service -- and I believe it needs to be a STRING.

I tighten up the API - such that the only way it can be set is VIA the new enumeration. I think this is a good compromise.

Reply:
yeah, I was thinking along those lines after I made the comment. I think that's the way to go